web-ui/src/main/resources/catalog/templates/admin/harvest/harvest-report.html

```diff
@@ -0,0 +1,30 @@
+
+  <div class="panel panel-default" data-ng-controller="GnHarvestReportController">
+    <div class="panel-heading">
+      <span data-translate="">reportHarvester</span>
+      <i data-ng-show="isLoadingHarvester" class="fa fa-spinner fa-spin"></i>
+      
+      <div class="btn-toolbar" data-ng-hide="$parent.isLoadingHarvester"></div>
+    </div>
+
+    <div class="panel-body">
+
+			<table class="table table-striped">
+				<tr>
+          <th data-translate="">harvesterName</th>
+          <th data-translate="">serviceUrl</th>
+          <th data-translate="">harvesterType</th>
+          <th data-translate="">lastRun</th>
+          <th data-translate="">totalNonTemplateMetadata</th>
+        </tr>
+        <tr
+          data-ng-repeat="h in $parent.harvesters">
+          <td>{{h.site.name | translate}}</td>
+          <td class="gn-break">{{(!h.site.url.length ? (!h.site.baseUrl.length : h.site.capabilitiesUrl : h.site.baseUrl) : h.site.url) | translate}}</td>
```

Comment:
Looks like the first ":" should be "?" and "| translate" does not sound useful here ?